### DISCUSSION TITLE
Extract AAC Audio Files From MP4 Videos (Not M4A)

### DISCUSSION THREAD

**Discussion by bytos** [mifi/lossless-cut#1871] [Q&A] [answered]
I want to cut very long Youtube videos and extract a portion of the AAC LC audio files, but not to a .m4a container file.

Could this option be added to LosslessCut since the app has many audio formats, except AAC.

**Comment by mifi** [comment]
choose output format "ipod"

**Reply by mifi** [reply]
AAC is not a format btw, it's a codec. so you probably want to put it in mp4 or m4a format

**Comment by bytos** [comment]
The output format is "ipod" and get .m4a files.

Have applications that work with .aac files, one of them doesn't accept .m4a files, only .aac files.

Wanted to save a step by getting .aac files from LosslessCut, but not the end of the world.

**Comment by mifi** [comment, marked as answer]
In such case it's probably ADTS (raw AAC), so you can choose that as an output format in losslesscut

<img width="268" alt="Screenshot 2024-01-28 at 10 45 34" src="https://github.com/mifi/lossless-cut/assets/402547/7cf4c50d-f6bd-4f73-a0e3-1518d805860f">

**Comment by bytos** [comment]
Initially, I thought I was getting .aac files from LosslessCut (older version), but have not touched the settings for a long time.

Since upgrading to new version, I didn't see/remember the ADTS AAC option in the list, thanks for pointing that out.
